java/server/src/org/openqa/grid/common/GridRole.java

```diff
@@ -40,4 +40,17 @@ public static GridRole get(String role) {
         return null;
     }
   }
+
+  public static String asString(GridRole role) {
```

Comment:
@lukeis Ah yea,makes sense. Thanks!  I changed the method to `@Override public String toString()` ... It's not used anywhere in the current code base, but seemed like a good addition. I'm happy to remove it, if you disagree.